README.md

```diff
@@ -0,0 +1,48 @@
+# Guitar API
+
+## Table of Contents
```

Comment:
Nitpick: Table of Contents aren’t really necessary since most savvy individuals will <kbd>Ctrl</kbd> + <kbd>F</kbd> for what they want.